openlibrary/macros/SearchResultsWork.html

```diff
@@ -1,4 +1,4 @@
-$def with (doc, decorations=None, cta=True, availability=None, extra=None, attrs=None, rating=None, reading_log=None, show_librarian_extras=False)
+$def with (doc, decorations=None, cta=True, availability=None, extra=None, attrs=None, rating=None, reading_log=None, show_librarian_extras=True)
```

Comment:
Oh we don't want this to _always_ be true ; we want this to be True when `SearchResultsWork` is rendered from the authors page! Search the code base for `SearchResultWork` and see if you can find which render call we should update :)

Reply:
Gotcha! Thanks for the feedback. I'm on it